cmd/argocd-repo-server/commands/argocd_repo_server.go

```diff
@@ -209,6 +211,7 @@ func NewCommand() *cobra.Command {
 	command.Flags().StringVar(&streamedManifestMaxExtractedSize, "streamed-manifest-max-extracted-size", env.StringFromEnv("ARGOCD_REPO_SERVER_STREAMED_MANIFEST_MAX_EXTRACTED_SIZE", "1G"), "Maximum size of streamed manifest archives when extracted")
 	command.Flags().StringVar(&helmManifestMaxExtractedSize, "helm-manifest-max-extracted-size", env.StringFromEnv("ARGOCD_REPO_SERVER_HELM_MANIFEST_MAX_EXTRACTED_SIZE", "1G"), "Maximum size of helm manifest archives when extracted")
 	command.Flags().BoolVar(&disableManifestMaxExtractedSize, "disable-helm-manifest-max-extracted-size", env.ParseBoolFromEnv("ARGOCD_REPO_SERVER_DISABLE_HELM_MANIFEST_MAX_EXTRACTED_SIZE", false), "Disable maximum size of helm manifest archives when extracted")
+	command.Flags().BoolVar(&preserveDependenciesChartsArchives, "preserve-dependencies-charts-archives", env.ParseBoolFromEnv("ARGOCD_REPO_PRESERVE_DEPENDENCIES_CHARTS_ARCHIVES", false), "Preserve dependencies charts archives during refresh")
```

Comment:
As a new environment variable is added, can you add the variable to repo-server deployment?

Reference: https://github.com/argoproj/argo-cd/blob/master/manifests/base/repo-server/argocd-repo-server-deployment.yaml#L26

Reply:
Thank you @ishitasequeira!
I added the requested change and also added to additional manifest files that have similar functionality - let me know if I should keep or remove them and stay only with the file requested by you.

Reply:
I removed that question as I just learned that the other files are auto generated